src/parser/xasl_generation.c

```diff
@@ -18019,6 +18028,7 @@ pt_to_insert_xasl (PARSER_CONTEXT * parser, PT_NODE * statement)
 	  (void) memcpy (xasl->class_oid_list, xasl_Supp_info.class_oid_list, sizeof (OID) * n);
 	  (void) memcpy (xasl->class_locks, xasl_Supp_info.class_locks, sizeof (int) * n);
 	  (void) memcpy (xasl->tcard_list, xasl_Supp_info.tcard_list, sizeof (int) * n);
+	  xasl->includes_tde_class = xasl_Supp_info.includes_tde_class;
```

Comment:
is guranteed xasl->includes_td_class not null ?

Reply:
`includes_tde_class` is not pointer